src/providers/mod.rs

```diff
@@ -155,6 +156,18 @@ pub trait Provide {
         Err(ResponseStatus::PsaErrorNotSupported)
     }
 
+    /// Lists all clients currently having data in the service.
+    fn list_clients(&self, _op: list_clients::Operation) -> Result<list_clients::Result> {
+        trace!("list_clients ingress");
+        Err(ResponseStatus::PsaErrorNotSupported)
+    }
+
+    /// Delete all data a client has in the service..
+    fn delete_client(&self, _op: delete_client::Operation) -> Result<delete_client::Result> {
+        trace!("delete_client ingress");
+        Err(ResponseStatus::PsaErrorNotSupported)
+    }
```

Comment:
As per #312 you should either not implement this or skip returning on `PsaErrorNotSupported` in `CoreProvider`, but I think you do neither

Reply:
If I don't do this, won't we have the same problem as explained in #312 then? As in I was thinking to do the same (bad) thing than `list_keys` and whatever solution is chosen for #312 would fix both of those things.

Reply:
Ok, that's fair, will add a comment about it in that issue